### DISCUSSION TITLE
Tile sensor severity colours

### DISCUSSION THREAD

**Discussion by tomlut** [home-assistant/frontend#14733] [Dashboard]
Can we get a few tile sensor colours based on numeric state, like the gauge card?

e.g. for a temperature sensor, red, blue, and green to make it obvious when the temperature is hot, cold or just right respectively.

**Comment by tomlut** [comment]
Look how much information this displays at a glance:

![Screenshot 2022-12-13 at 16-39-39 New Test – Home Assistant](https://user-images.githubusercontent.com/10679300/207284486-23a1ba3b-8964-4e79-aeef-9976ff75fd08.png)

(done with card_mod).

**Comment by tomlut** [comment]
```
        card_mod:
          style: |
            .tile {
              {% if states(config.entity)|float(0) > 24.9 %}
                --tile-color: var(--rgb-red-color);
              {% elif states(config.entity)|float(0) > 22.9 %}
                --tile-color: var(--rgb-yellow-color);
              {% elif states(config.entity)|float(0) > 17.9 %}  
                --tile-color: var(--rgb-green-color);
              {% else %}  
                --tile-color: var(--rgb-blue-color);
              {% endif %}  
            }
```

**Comment by bkbilly-intrack** [comment]
Thanks for the suggestion, it looks really nice, though I still believe it would be nice to have a gauge because it can be used with more applications.
Maybe an integration with the custom card [bar-card](https://github.com/custom-cards/bar-card) would be nice.

**Comment by ChristophCaina** [comment]
The approach from tomlut is looking good - and it would meet the design approaches for the "next - gen" UI... :)
The Gauge-Card does not - and I beleive, the dev team doesn't really care about "custom bar card"...

While I agree, that a Gauge is a very helpfull card, it does not really provide much information at the first glance.
Not, if you have multiple gauges in the same area nearby... And I know what I am talking after creating Dashboards with aprox 9 or 10 gauge meters ... (was a silly approach :D )

Personally, what I'm more interested in:
How did you get the tile cards into the other card?
Is that a grid-stack?

**Comment by tomlut** [comment]
Hui-element. See the write-up here: https://community.home-assistant.io/t/icon-color-on-themes-open-window/499940/7

**Reply by ChristophCaina** [reply]
thanks - I've overlooked the "square: false" parameter :D

**Reply by ChristophCaina** [reply]
and the custom hui element ...

**Reply by ChristophCaina** [reply]
thanks :)
yes, that's something I can work & live with... when this will be possible without using "custom" components - I think that would be really a great addition to the dashboard elements :)
![grafik](https://user-images.githubusercontent.com/26391061/208056112-c0f890ad-c043-48c3-8d72-98a9b897c9f8.png)
